app/views/foods/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<section>
+  <div class="row justify-content-center">
+    <div class="col-md-6">
+      <div class="form-container">
+        <h2 class="form-title">Add New Food</h2>
+          <%= render "form", food: @food %>
+      </div>
+    </div>
+  </div>
+</section>
```

Comment:
- [Optional] Hi @team, I think it could have been better to validate the user's input and display error messages when a user attempt to submit the form without filling out the fields completely. This could improve the user's experience while using your app.